src/projects/NewProjectPage.tsx

```diff
@@ -1,10 +1,25 @@
 import React, { useState } from "react";
 import { useRouter } from "next/router";
+import axios, { AxiosRequestConfig } from "axios";
 
 import { styled } from "../stitches.config";
 
-const callApi = (receiverName: string) => {
-  return "abcdefg";
+type CreateProjectResponse = {
+  project_id: string;
+};
+
+const createProjectApiUrl = "https://api/projects";
+
+const createNewProject = (receiverName: string) => {
+  async () => {
+    const requestConfig: AxiosRequestConfig = {
+      url: createProjectApiUrl,
+      method: "POST",
+      data: receiverName,
```

Comment:
```suggestion
      data: {
        receiverName: receiverName
      },
```

じゃない？

```ts
type CreateProjectApiRequest  = {
  receiver_name: string;
}
const requestConfig: AxiosRequestConfig<CreateProjectApiRequest> ={ ... }
```

みたいに型を定義してあげるといい感じかな